l10n_th_address/__manifest__.py

```diff
@@ -32,6 +32,5 @@
     ],
     'installable': True,
     'auto_install': False,
+    'post_init_hook': '_update_data_noupdate',
```

Comment:
Please add the OCA to the `author` key (check the OCA module template in OCA/maintainer-tools).